### DISCUSSION TITLE
STM32U5 using LPTIM (Timeout function)

### DISCUSSION THREAD

**Discussion by Joachim850** [zephyrproject-rtos/zephyr#75096] [Q&A]
I want to use the LPTIM1 on STM32U575 to implement a timeout function. 
Is that possible with the zephyr driver "stm32_lptim_timer.c"?

How do I configure this?

**Comment by erwango** [comment]
You'd need a dedicated drivet to do this. `stm32_lptim_timer.c` implements timer api which is used to provide ticks to kernel.
